shapeworks_cloud/manage_workers.py

```diff
@@ -1,9 +1,15 @@
+import datetime
 import os
+from pathlib import Path
 import sys
 import time
 
 import boto3
 
+# DEPLOY_LOCK = Path('/opt/django-project/dev/deploy.lock')  # for development
+DEPLOY_LOCK = Path('/home/ubuntu/celery_project/dev/deploy.lock')
```

Comment:
Should we instead use a path relative to the source file?  Maybe `Path(__file__) / '..' / 'deploy.lock'`?

Reply:
Changed in [db7789a](https://github.com/SCIInstitute/shapeworks-cloud/pull/378/commits/db7789a5a37a9e9447dd51b45916ff292d6fb18b)